frontend/src/components/authentication/AuthenticationTabs.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import TabNavigation from "../commons/TabNavigation";
+import LoginForm from "./LoginForm";
+import CreateAccountForm from "./CreateAccountForm";
+import LOCALIZE from "../../text_resources";
+
+const customStyles = {
+  container: {
+    width: 500,
+    margin: "0px auto",
+    paddingTop: 24,
+    paddingRight: 20,
+    paddingLeft: 20
+  }
+};
+
+class LoginTabs extends Component {
+  //TODO(fnormand): Remove this part when implementing login functionality in the backend
+  //===========================================
+  static propTypes = {
+    authentification: PropTypes.func
+  };
+
+  state = {
+    isAuthenticated: false
+  };
+
+  authentification = () => {
+    this.setState({ isAuthenticated: true });
+    this.props.authentification();
+  };
+  //===========================================
+
+  render() {
+    const TABS = [
+      {
+        id: 0,
+        tabName: LOCALIZE.homePage.login.title,
```

Comment:
(Comment Ref 1) Since the login won't live on the home page necessarily, I'm thinking this should be 'LOCALIZE.login' and then maybe have '.autheniticationTabs', '.createAccount' '.loginForm' underneath it

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/152/commits/6ce81fd2552bf6633dd7149f12e43f64df4e3db2